packages/lesswrong/components/votes/SmallSideVote.tsx

```diff
@@ -36,30 +38,40 @@ const styles = theme => ({
   }
 })
 
-const CommentsVote = ({ comment, hideKarma=false, classes }: {
-  comment: CommentsList,
+const SmallSideVote = ({ document, hideKarma=false, classes, collection }: {
+  document: CommentsList|PostsList,
   hideKarma?: boolean,
   classes: ClassesType,
+  collection: any
 }) => {
   const currentUser = useCurrentUser();
   const vote = useVote();
   const {eventHandlers, hover} = useHover();
   
-  if (!comment) return null;
+  if (!document) return null;
 
   const { VoteButton } = Components
-  const voteCount = comment.voteCount;
-  const karma = Comments.getKarma(comment)
+  const voteCount = document.voteCount;
+  let karma = 0
+  let moveToAlignnmentUserId = ""
+  if (collection == Posts) {
```

Comment:
Double-equal should almost always be avoided. A triple-equal should get you the same result here.